netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -173,8 +178,19 @@ private static Http2Headers convertMetadata(Metadata headers) {
 
     @Override
     public EventLoopGroup create() {
-      return new NioEventLoopGroup(nEventLoops, new ThreadFactoryBuilder().setNameFormat(name + "-%d")
-          .build());
+      // Use the executor based constructor so we can work with both Netty4 & Netty5.
+      ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat(name + "-%d").build();
+      int parallelism = nEventLoops == 0 ?
```

Comment:
@louiscryan we don't that line anymore, the nEventLoops == 0 case is now part of the offical Netty API [1]. It's in 5 already, still have to submit a PR for 4.0 and 4.1, but will do today or monday.

https://github.com/netty/netty/commit/93bcc6bbdef53e9249802e1bba72aca06f90ad45